apps/src/templates/sectionAssessments/CorrectAnswerIcon.jsx

```diff
@@ -1,19 +1,54 @@
-import React, {Component} from 'react';
+import React, {Component, PropTypes} from 'react';
 import color from "@cdo/apps/util/color";
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
 
 const styles = {
+  main: {
+    border: 'none',
```

Comment:
nit: `height: 30` is preferred over `height: '30px'`
styleguide: https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#react

Note: this only works for numbers, not percentages (which need to remain strings)

Reply:
Fixed, thanks.